internal/tiltfile/tiltfile_state.go

```diff
@@ -1226,6 +1235,34 @@ func (s *tiltfileState) setTeam(thread *starlark.Thread, fn *starlark.Builtin, a
 	return starlark.None, nil
 }
 
+func (s *tiltfileState) dockerPruneSettings(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
```

Comment:
all new functions should be implemented as extensions, see this pr for an example:
https://github.com/windmilleng/tilt/pull/2330

it should be pretty straightfoward

Reply:
will do in a separate PR to reduce size of this one